modules/learn/pages/clusters-and-availability/nodes.adoc

```diff
@@ -150,14 +149,25 @@ Fast failover can be achieved by deploying an arbiter node, which is a node that
 Such a deployment ensures that the Cluster Manager automatically deploys critical system infrastructure on the arbiter node, rather than on any other, so as to avoid clashes.
 This may lead to several seconds of latency reduction, when a busy cluster undergoes failover.
 
-For information about service deployment with:
+=== Deploying Services on a Node
 
-* The REST API, see xref:rest-api:rest-set-up-services.adoc[Assigning Services].
+You can add services to a new node when adding the node to a cluster.
+
+You can also add or remove non-Data Services on an existing node in a cluster, without adding or removing nodes.
+
+For information about the deployment of services on a new node using:
+
+* The REST API, see xref:rest-api:rest-set-up-services.adoc[Assigning Services to a New Node].
 
 * Couchbase Server Web Console during node-addition and node-joining, see the demonstrated uses of checkboxes, in xref:manage:manage-nodes/add-node-and-rebalance#arbiter-node-addition[Add a Node and Rebalance] and in xref:manage:manage-nodes/join-cluster-and-rebalance#arbiter-node-join[Join a Cluster and Rebalance].
 
 * The CLI during node-addition, see xref:cli:cbcli/couchbase-cli-server-add[server-add].
 
+For information about adding or removing non-Data Services on an existing node in a cluster, using:
+
+* The xref:manage:manage-nodes/modify-services-on-nodes-and-rebalance.adoc#modify-mds-services-from-ui[UI] and xref:manage:manage-nodes/modify-services-on-nodes-and-rebalance.adoc#modify-mds-services-using-cli[CLI], see xref:manage:manage-nodes/modify-services-and-rebalance.adoc[Modify Services and Rebalance].
+* The REST API, see xref:rest-api:rest-set-up-services-existing-nodes.adoc[Assigning Services to an Existing Node].
```

Comment:
`* The REST API, see xref:rest-api:rest-set-up-services-existing-nodes.adoc[Assigning Services to an Existing Cluster Node].`